db/migrate/20200814034110_add_neighborhood_to_locations.rb

```diff
@@ -0,0 +1,5 @@
+class AddNeighborhoodToLocations < ActiveRecord::Migration[6.0]
+  def change
+    add_column :locations, :neighborhood, :string
```

Comment:
Should we also add borough while we're at it? I know that we sort of sub city for that right now but it's not quite accurate

Reply:
not a blocker

Reply:
I guess we can also just use city as borough like we're already doing

Reply:
Yes good call. I did notice that we're using the "City" field to represent borough, which is not scalable lol. Will have a followup PR out for that soon.